scipy/stats/__init__.py

```diff
@@ -237,6 +237,7 @@
    tsem              --
    variation         -- Coefficient of variation
    find_repeats
+   rankdata
```

Comment:
`tiecorrect` computes something given ranks from `rankdata`, so this is a reasonable place.
```suggestion
   rankdata
   tiecorrect
```